cpp/include/cudf/binaryop.hpp

```diff
@@ -214,5 +214,47 @@ cudf::data_type binary_operation_fixed_point_output_type(binary_operator op,
                                                          cudf::data_type const& lhs,
                                                          cudf::data_type const& rhs);
 
+namespace binops {
+
+/**
+ * @brief Computes output valid mask for op between a column and a scalar
+ *
+ * @param col     Column to compute the valid mask from
+ * @param s       Scalar to compute the valid mask from
+ * @param stream  CUDA stream used for device memory operations and kernel launches
+ * @param mr      Device memory resource used to allocate the returned valid mask
+ * @return        Computed validity mask
+ */
+rmm::device_buffer scalar_col_valid_mask_and(
```

Comment:
There is `bitmask_and` API in cudf so instead of `_valid_mask_and` this should be better called `_bitmask_and` for consistentcy.

Reply:
In addition, I'm a bit confused with this function. bitmask and between a column and a scalar? If the scalar is null, is the result be all nulls? If the scalar is not null, is the result the same as the input column's bitmask?